src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -350,11 +350,13 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
             impl_desc_type::jit_avx512_amx,
             impl_desc_type::brgconv_avx512_1x1,
             impl_desc_type::brgconv_avx512,
+            impl_desc_type::brgconv_avx512_dw,
```

Comment:
Hi, @tiger100256-hu : based on oneDNN conv kernel [priority list](https://github.com/oneapi-src/oneDNN/blob/edc1d8cbeca4ef722e81f81656dd951d2a65b88b/src/cpu/cpu_convolution_list.cpp#L123-L125) (And for some models 'brgconv_avx512_amx' shows less efficient than brdgmm, e.g. in ticket CVS-157009), I think we should put this new brdgmm kernel before 'impl_desc_type::brgconv_avx512_amx_1x1' in this list.

BTW,  does this kind of new 'impl_desc_type::brdgmm_avx512_dw' name seems better(to align with oneDNN kernel name)?

Reply:
@liubo-intel 
1. about priority, I will update code to give brdgmm higher priority 
2. about f32 bias can't select this kernel when run with precison f16, let us discuss with @dmitry-gorokhov  in ticket  [CVS-157009](https://jira.devtools.intel.com/browse/CVS-157009)
3. about the 'impl_desc_type::brdgmm_avx512_dw' , @luweizhou2016 how do you think? I am ok with any of the two names brgdmm_avx512_dw or brgconv_avx512_dw